base_layer/wallet/src/lib.rs

```diff
@@ -54,3 +55,10 @@ pub type WalletSqlite = Wallet<
     ContactsServiceSqliteDatabase,
     KeyManagerSqliteDatabase,
 >;
+
+hash_domain!(
+    WalletSecretKeysDomain,
+    "com.tari.tari_project.base_layer.wallet.secret_keys",
```

Comment:
Why add `tari_project` to the hash domain?

Reply:
This is to differentiate between different GitHub projects.